README.md

```diff
@@ -1,19 +1,26 @@
+[integration-tests-badge]: https://github.com/conda-incubator/conda-recipe-manager/actions/workflows/integration_tests.yaml/badge.svg?branch=main
+[release-badge]: https://img.shields.io/github/v/release/conda-incubator/conda-recipe-manager?logo=github
+
+
 # conda-recipe-manager
 
+[![Integration Tests][integration-tests-badge]](https://github.com/conda-incubator/conda-recipe-manager/actions/workflows/integration_tests.yaml)
+[![GitHub Release][release-badge]](https://github.com/conda-incubator/conda-recipe-manager/releases)
+
 ## Table of Contents
 <!-- TOC -->
 
 - [conda-recipe-manager](#conda-recipe-manager)
-    - [Table of Contents](#table-of-contents)
+  - [Table of Contents](#table-of-contents)
```

Comment:
This table of contents is auto-generated by a VS Code plugin (hence the `<-- TOC -->`). I don't think these indents will break it, but I think we will generate another diff later.

I'm guessing you have a tool that formats `README`s and that's now conflicting with that TOC plugin(?)

Reply:
It might be a specific VSCode extension; I can't seem to save the file without these new spaces being added. Will re-commit a non-edited version.